dsp_permissions_scripts/ap/ap_set.py

```diff
@@ -36,3 +40,26 @@ def apply_updated_scopes_of_aps_on_server(aps: list[Ap], host: str, dsp_client:
         except ApiError as err:
             logger.error(err)
     logger.info(f"Finished updating scopes of {len(aps)} Administrative Permissions on {host}")
+
+
+def create_new_ap_on_server(
+    forGroup: Group,
+    shortcode: str,
+    hasPermissions: list[ApValue],
+    dsp_client: DspClient,
+) -> Ap | None:
+    proj_iri, _ = get_project_iri_and_onto_iris_by_shortcode(shortcode, dsp_client)
+    payload = {
+        "forGroup": forGroup.val.replace("knora-admin:", KNORA_ADMIN_ONTO_NAMESPACE),
```

Comment:
cosmetic remark: this logic looks like it could be nicerly hidden in a function like `forGroup.expand_namespace()`